package.json

```diff
@@ -4,20 +4,21 @@
     "license": "MIT",
     "scripts": {
         "ng": "ng",
+        "ng-prefer-ipv4": "node --dns-result-order ipv4first ./node_modules/@angular/cli/bin/ng",
         "build": "npm run build-prod:core && npm run build-apps",
         "build:core": "ng build --configuration development core",
         "build-watch:core": "ng build --configuration development core --watch",
         "build-prod:core": "ng build --configuration production core",
         "build-apps": "npm run build-prod:gis && npm run build-prod:dashboards/data-atlas && npm run build-prod:dashboards/ebv-analyzer && npm run build-prod:dashboards/gfbio && npm run build-prod:dashboards/nfdi-gdo",
-        "serve:dashboards/data-atlas:local": "ng serve --configuration development data-atlas --proxy-config proxy-local.conf.json",
+        "serve:dashboards/data-atlas:local": "npm run ng-prefer-ipv4 -- serve --configuration development data-atlas --proxy-config proxy-local.conf.json",
```

Comment:
That works :+1: .

I guess we can make this even better:

```
"ng-serve-local": "node --dns-result-order ipv4first ./node_modules/@angular/cli/bin/ng serve --configuration development $1 --proxy-config proxy-local.conf.json",
"serve:gis:local": "npm run ng-serve-local gis",
```

Reply:
Done in https://github.com/geo-engine/geoengine-ui/pull/511/commits/75ca64dac66cb755d486da9707d0c4d9ac50cf89.
The "$1" does not do anything however, because arguments are always simply appended. So I left it out and changed the order of the arguments in the helper script.
(There exists a workaround which allows for positional arguments but it relies on shell, so leads to issues on Windows cmd apparently.)